thirdeye/docs/application.rst

```diff
@@ -0,0 +1,96 @@
+..
+.. Licensed to the Apache Software Foundation (ASF) under one
+.. or more contributor license agreements.  See the NOTICE file
+.. distributed with this work for additional information
+.. regarding copyright ownership.  The ASF licenses this file
+.. to you under the Apache License, Version 2.0 (the
+.. "License"); you may not use this file except in compliance
+.. with the License.  You may obtain a copy of the License at
+..
+..   http://www.apache.org/licenses/LICENSE-2.0
+..
+.. Unless required by applicable law or agreed to in writing,
+.. software distributed under the License is distributed on an
+.. "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+.. KIND, either express or implied.  See the License for the
+.. specific language governing permissions and limitations
+.. under the License.
+..
+
+.. _application:
+
+****************************
+Application
+****************************
+
+An application is a basic TE entity can serves as a container for metrics, alerts and other entities.
+It also can be used to group a bunch of users.
+
+Creating an Application
+##########################################
+
+You can create an Application in ThirdEye using 2 ways:
+1. ThirdEye Admin
+2. Using the API
```

Comment:
Right now, the admin portal doesn't have the capability to add a new application for the first time. We need to create an application using the API first and subsequently, we can create more applications from the portal. This is a minor UI bug.

Issue: https://github.com/apache/incubator-pinot/issues/5525

Reply:
Hey @akshayrai I'm able to create an application from the UI. In fact, I was documenting this while I was going through the process.

Reply:
Awesome! Looks like it was added recently: https://github.com/apache/incubator-pinot/pull/5601